src/bp-messages/bp-messages-template.php

```diff
@@ -2242,3 +2242,45 @@ function bp_embed_message_cache( $cache, $id, $cachekey ) {
 function bp_embed_message_save_cache( $cache, $cachekey, $id ) {
 	bp_messages_update_meta( $id, $cachekey, $cache );
 }
+
+
+
+/**
+ * Get the total number of recipients in the current thread.
+ *
+ * @since x.x.x
+ *
+ * @return int
+ */
+function bp_get_thread_total_recipients_count() {
+	global $thread_template;
+	/**
+	 * Filters the total number of recipients in a thread.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param int $count Total recipients number.
+	 */
+	return (int) apply_filters( 'bp_get_thread_total_recipients_count', $thread_template->thread->total_recipients_count );
+}
+
+/**
+ * Get all admin users.
+ * 
+ * @since x.x.x
+ * 
+ * @return array
+ */
+function bb_get_all_admin_user() {
+	$args = array(
+		'role'    => 'administrator',
+		'orderby' => 'user_nicename',
+		'order'   => 'ASC',
+		'fields'  => 'id',
+	);
+	$users = get_users( $args );
+	if ( ! empty( $users ) ) {
+		$users = array_map( 'intval',$users );
+	}
+	return $users;
+}
```

Comment:
```suggestion
/**
 * Get all admin users.
 * 
 * @since BuddyBoss 1.7.6
 * 
 * @return array
 */
function bb_get_all_admin_user() {
	$args = array(
		'role'    => 'administrator',
		'orderby' => 'user_nicename',
		'order'   => 'ASC',
		'fields'  => 'id',
	);
	$users = get_users( $args );
	if ( ! empty( $users ) ) {
		$users = array_map( 'intval',$users );
	}
	return $users;
}
```

Move to this function in core functions.php file